### DISCUSSION TITLE
Projects beta: allowing for multiple fields sort

### DISCUSSION THREAD

**Discussion by pragmanomos** [community/community#12390] [Projects and Issues] [answered]
Sometimes it's very helpful to sort issues by a cascade of multiple columns (e.g., sort by assignee first, status second etc.). Currently it's only possible to sort items based on one attribute.

**Comment by corey-synapse** [comment]
Plus one. Please!

**Reply by forvenus** [reply]
yes, please!

**Reply by MaximKotliar** [reply]
Yaaas

**Reply by grymmy** [reply]
Please add this feature, it's a no-brainer!

**Comment by rileybroughten** [comment, marked as answer]
👋 Closing in favor of https://github.com/orgs/community/discussions/8895 👍
